include/public/ingenialink/net.h

```diff
@@ -57,6 +57,34 @@ typedef enum {
 	IL_NET_PROT_VIRTUAL,
 } il_net_prot_t;
 
+enum update_error
+{
+	UP_NOERROR = 1,
+	UP_STATEMACHINE_ERROR = -32,
+	UP_NOT_IN_BOOT_ERROR = -33,
+	UP_EEPROM_PDI_ERROR = -34,
+	UP_EEPROM_FILE_ERROR = -35,
+	UP_NOT_FOUND_ERROR = -36,
+	UP_NO_SOCKET = -37,
+	UP_FORCE_BOOT_ERROR = -38
+};
+
+/** SOEM Error types */
+enum soem_ec_err_type
```

Comment:
Why redeclare this enum? This is already on ethercattype.h

If its really needed here can't you import it?
```
// Enum on ethercat types
typedef enum : uint8_t
{
    u8EData_Rate_3 = (uint8_t)1
} u8EDRATE;

// Enum on net
enum : uint8_t
{
    hello = u8EData_Rate_3
} example;
```